aiohttp/compression_utils.py

```diff
@@ -111,8 +111,8 @@ async def decompress(self, data: bytes, max_length: int = 0) -> bytes:
             self._max_sync_chunk_size is not None
             and len(data) > self._max_sync_chunk_size
         ):
-            return await asyncio.get_event_loop().run_in_executor(
-                self._executor, self.decompress_sync, data, max_length
+            return await asyncio.get_running_loop().run_in_executor(
```

Comment:
This code looks like its unreachable because we never create ZLibDecompressor with max_sync_chunk_size